lib/rules/padding-line-between-tags.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * @author dev1437
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const utils = require('../utils')
+
+/**
+ * Split the source code into multiple lines based on the line delimiters.
+ * Copied from padding-line-between-blocks
+ * @param {string} text Source code as a string.
+ * @returns {string[]} Array of source code lines.
+ */
+function splitLines(text) {
+  return text.split(/\r\n|[\r\n\u2028\u2029]/gu)
+}
+
+/**
+ * @param {RuleContext} context
+ * @param {VElement} tag
+ * @param {VElement} sibling
+ */
+function insertNewLine(context, tag, sibling) {
+  context.report({
+    messageId: 'always',
+    loc: sibling.loc,
+    // @ts-ignore
+    fix(fixer) {
+      return fixer.insertTextAfter(tag, '\n')
+    }
+  })
+}
+
+/**
+ * @param {RuleContext} context
+ * @param {VEndTag | VStartTag} endTag
+ * @param {VElement} sibling
+ */
+function removeExcessLines(context, endTag, sibling) {
+  context.report({
+    messageId: 'never',
+    loc: sibling.loc,
+    // @ts-ignore
+    fix(fixer) {
+      const start = endTag.range[1]
+      const end = sibling.range[0]
+      const paddingText = context.getSourceCode().text.slice(start, end)
+      const lastSpaces = splitLines(paddingText).pop()
+      return fixer.replaceTextRange([start, end], `\n${lastSpaces}`)
```

Comment:
I've been testing this and noticed a loss of strings.

```vue
<template>
  <div>
    </div>A
    B
    C<div />
  </div>
</template>

<script>
/* eslint "vue/padding-line-between-tags": ["error", [
    { "blankLine": "never", "prev": "*", "next": "*" }
  ]]
*/
</script>
```

↓

```vue
<template>
  <div>
    </div>
    C<div />
  </div>
</template>

<script>
/* eslint "vue/padding-line-between-tags": ["error", [
    { "blankLine": "never", "prev": "*", "next": "*" }
  ]]
*/
</script>
```

Perhaps it makes more sense to exclude from the check if there is anything other than element between tags.
What do you think?